ts/data/case.ts

```diff
@@ -184,6 +189,25 @@ export const caseDefinition: TableDefinition<Case, Test> = {
               (kase.timepoint ? " " + kase.timepoint : "")
           )
         );
+        const tooltipDiv = document.createElement("div");
+        const tissueOriginDiv = document.createElement("div");
+        tissueOriginDiv.appendChild(
+          document.createTextNode(`Tissue Origin: ${kase.tissueOrigin}`)
+        );
+        tooltipDiv.appendChild(tissueOriginDiv);
+        const tissueTypeDiv = document.createElement("div");
+        tissueTypeDiv.appendChild(
+          document.createTextNode(`Tissue Type: ${kase.tissueType}`)
+        );
+        tooltipDiv.appendChild(tissueTypeDiv);
+        const timepointDiv = document.createElement("div");
+        timepointDiv.appendChild(
+          document.createTextNode(
+            kase.timepoint ? `\nTimepoint: ${kase.timepoint}` : ""
+          )
+        );
+        tooltipDiv.appendChild(timepointDiv);
```

Comment:
Check whether the case has a timepoint and only create the timepoint div if so.

Can also create an `addTextDiv(text, container)` function that creates the div, sets the text, and adds it to the container (`tooltipDiv`) to reduce the repetition